src/Utilities.gs

```diff
@@ -32,30 +32,40 @@ function buildUrl_(url, params) {
 
 /**
  * Validates that all of the values in the object are non-empty. If an empty
- * value is found, an error is thrown using the key as the name.
+ * value is found, and error is thrown using the key as the name.
  * @param {Object.<string, string>} params The values to validate.
  * @private
  */
 function validate_(params) {
   Object.keys(params).forEach(function(name) {
     var value = params[name];
-    if (isEmpty_(value)) {
-      throw Utilities.formatString('%s is required.', name);
+    if (!value) {
```

Comment:
Not exactly the same as isEmpty (empty array different than empty string), but so  long as value is always a string then it's fine.

Reply:
Yes, but we did the same in the OAuth2 lib a while back and it didn't seem to have any negative consequences. This helper is just there to warn the user about missing settings, so a false negative here just means a less clear error message.